tests/quotes/strings.js

```diff
@@ -31,6 +31,22 @@
 // Unnecessary escapes.
```

Comment:
Good call, I cherry-picked the relevant change over from https://github.com/prettier/prettier/pull/1571. Once this PR has stabilized, I can port the tests over to #1571, since that's where we're actually making sure that directives don't get transformed.